swagger.yaml

```diff
@@ -765,6 +765,824 @@ definitions:
         type: integer
 
 
+  User:
+    description: "The user model for the response body"
+    type: object
+    properties:
+      id:
+        description: "The users id"
+        type: string
+      forename:
+        description: "The users forename"
+        type: string
+      surname:
+        description: "The users surname"
+        type: string
+      email:
+        description: "The users email address"
+        type: string
+      groups:
+        description: "The groups the user belongs to"
+        type: array
+        items:
+          description: "The groups name"
+          type: string
+      active:
+        description: "Whether or not the user is active"
+        type: boolean
+      status_notes:
+        description: "Notes about the updates made to the user"
+        type: string
+        example: "User has been suspended"
+      status:
+        description: "The current status of the user"
+        type: string
+        enum: [UNCONFIRMED, CONFIRMED, ARCHIVED, COMPROMISED, UNKNOWN, RESET_REQUIRED, FORCE_CHANGE_PASSWORD]
+        example: "CONFIRMED"
+
+  ErrorResponse:
+    description: "A list of any errors"
+    type: object
+    properties:
+      errors:
+        type: array
+        items:
+          $ref: '#/definitions/ErrorObject'
+
+  ErrorObject:
+    description: "An individual error details"
+    type: object
+    properties:
+      code:
+        type: string
+        description: "Code representing the type of error that occurred"
+swagger: "2.0"
+info:
+  title: dp-identity-api
+  description: "An API used to manage the authorisation of users accessing data publishing services."
+  version: 1.0.0
+  license:
+    name: "Open Government Licence v3.0"
+    url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+basePath: "/v1"
+schemes:
+  - http
+paths:
```

Comment:
There seems to be something going on with this swagger spec as the header has been duplicated. Maybe a merge conflict issue?